src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -198,8 +198,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // Set the foreground and background here in the constructor
         // (they will also be called later in UpdateAppearance, but we want to have it here so
         //  that the terminal is not gray on startup)
-        const auto bg = _settings.DefaultBackground();
-        _BackgroundColorChanged(bg);
+        til::color newBgColor{ _settings.DefaultBackground() };

```

Comment:
i suspect you know what i am going to say about us having three copies of this code ;P

Reply:
ahh haha alright that's fair... I was just trying to have it so there's only 1 update appearance call but okay will make a helper for the fg/bg